BlazarService/src/main/java/com/hubspot/blazar/listener/SlackDmNotificationVisitor.java

```diff
@@ -44,6 +44,10 @@ public SlackDmNotificationVisitor(BlazarConfiguration blazarConfiguration,
 
   @Override
   public void visit(RepositoryBuild build) throws Exception {
+    if (!build.getState().isComplete()) {
```

Comment:
This was required and missing. Otherwise we try to get fields from the RepoBuild that are not filled out because the build is in an early state (Queued etc).